crates/re_types/build.rs

```diff
@@ -33,41 +34,28 @@ fn main() {
         return;
     }
 
-    rerun_if_changed_or_doesnt_exist(SOURCE_HASH_PATH);
+    // Only re-build if source-hash exists
+    if !<str as AsRef<Path>>::as_ref(SOURCE_HASH_PATH).exists() {
```

Comment:
```suggestion
    if !Path::from(SOURCE_HASH_PATH).exists() {
```

Reply:
That was the first thing I tried, but I got:
```
the size for values of type `[u8]` cannot be known at compilation time
within `std::path::Path`, the trait `std::marker::Sized` is not implemented for `[u8]`
```

Reply:
Aha, I wanted `Path::new`